### ISSUE TITLE
Question about your site and new Giiker i3S + i3SE compatibility 

### ISSUE THREAD

**Issue by ElenaLauder** [tao-yu/Alg-Trainer#9]
Hi,
I hope you are well. I would like to memorize F2L alg's and I have read you site is good for learning.
I can connect the cube, but I think my moves are not reflected on the virtual cube.
My moves makes no sense with the moves on the virtual cube on your. Doe's I made something wrong?
Thank you in advance for a respond.
Kind Regards, 
Elena

**Comment by tao-yu** [comment]
> Hi,
> I hope you are well. I would like to memorize F2L alg's and I have read you site is good for learning.
> I can connect the cube, but I think my moves are not reflected on the virtual cube.
> My moves makes no sense with the moves on the virtual cube on your. Doe's I made something wrong?
> Thank you in advance for a respond.
> Kind Regards,
> Elena

Hi there. Sorry that my site isn't working for you. I have been made aware of this issue by one other person and I think it is likely due to the changed software on the new Giiker i3s. I think there are several websites such as cstimer which have found ways to fix this problem, so I definitely want to look into this as soon as I can. 

I'm quite busy due to college at the moment, so the earliest I can guarantee having time to investigate this is early May, although I'll try and put some time aside in early April to see if the fix is possibly easier than I expected.

Again, sorry for any inconvenience caused.

**Comment by ElenaLauder** [comment]
Hi Tao-Yu,

Thank you for your fast respond.

The first carry for yourself!

I respect that. Easy, no problem.

 

I found a site, I have checked my cube and this is showing.

 

 <https://briefcubing.com/experimental/giiker2/> https://briefcubing.com/experimental/giiker2/

My Giiker i3SE is showing:

 

0000aadc-0000-1000-8000-00805f9b34fb

1234567833333333123456789101112000013111515152

 

My Giiker i3S is showing:

0000aadc-0000-1000-8000-00805f9b34fb

1234567833333333123456789101112000063611013913

 

Maybe this can help.

Thank you in advance for your hard work and effort.

Kind Regards,

Elena

 

 

Von: tao-yu <notifications@github.com> 
Gesendet: Mittwoch, 25. März 2020 00:28
An: tao-yu/Alg-Trainer <Alg-Trainer@noreply.github.com>
Cc: ElenaLauder <elena.lauder@gmx.net>; Author <author@noreply.github.com>
Betreff: Re: [tao-yu/Alg-Trainer] Question about your site and new Giiker i3S + i3SE compatibility (#9)

 

Hi there. Sorry that my site isn't working for you. I have been made aware of this issue by one other person and I think it is likely due to the changed software on the new Giiker i3s. I think there are several websites such as cstimer which have found ways to fix this problem, so I definitely want to look into this as soon as I can.

I'm quite busy due to college at the moment, so the earliest I can guarantee having time to investigate this is early May, although I'll try and put some time aside in early April to see if the fix is possibly easier than I expected.

Again, sorry for any inconvenience caused.
